src/python/pants/option/global_options.py

```diff
@@ -295,6 +298,9 @@ def register_bootstrap_options(cls, register):
     register('--remote-store-chunk-upload-timeout-seconds', type=int, advanced=True,
              default=DEFAULT_EXECUTION_OPTIONS.remote_store_chunk_upload_timeout_seconds,
              help='Timeout (in seconds) for uploads of individual chunks to the remote file store.')
+    register('--remote-execution-cache-key-gen-version', advanced=True,
```

Comment:
This might be an opportunity to shorten this name, and have the help refer to the previous name... maybe `--remote-execution-cache-namespace`?

And given that this will only affect process invokes (and not CAS uploads), maybe `--remote-execution-process-cache-namespace` (or whatever the remote execution API calls this cache).